app/uk/gov/hmrc/gvmshaulierstatus/repositories/HaulierStatusRepository.scala

```diff
@@ -71,10 +71,10 @@ class HaulierStatusRepository @Inject()(
         .map(_.map(_.id))
     )
 
-  def create(correlationId: CorrelationId): Future[String] =
+  def create(correlationId: CorrelationId)(implicit instant: Instant = Instant.now(Clock.systemUTC())): Future[String] =
```

Comment:
do we need implicit here ? i see we only pass this param for case class
either pass the instant param along with correlationId or we can have apply(id) function in the case class that just creates instant value internally ?

Reply:
lets keep it in the arg list then, if not, not a biggie..